### ISSUE TITLE
Resize left column

### ISSUE THREAD

**Issue by mediapinta** [sonnyp/Tangram#241]
Hi,

Tangram is a great idea, and works! I don't have to keep opened tabs and tabs in my browser and clients and everything, just one program with all together (mails, chats, social networks, RSS reader...).

Here a suggestion: being able to resize the left column. I would like to have full space for the main column and also now the custom name of the apps get cut after some length. To me would be enough to keep small icons in the left column (maybe add hover info for the icons...)

Thank you!

Greetings,
ifrit

**Comment by Or4ma** [comment]
Yeah I was surprised that something like that isn't supported. I'v switched back to Ferdium because of this feature.
at least having the ability to hide the left column would make everything better.
